exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -54,6 +54,23 @@ public void testConcatOnNonEmptyListOfLists() {
                 ListOps.concat(listOfLists));
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testConcatOnNonEmptyListOfNestedLists() {
+        List<List<List<Character>>> listOfNestedLists = Arrays.asList(
+                Arrays.asList(Collections.singletonList('1'),
+                Collections.singletonList('2')),
+                Arrays.asList(Collections.singletonList('3'),
+                Collections.emptyList()),
+                Arrays.asList(Arrays.asList('4', '5', '6'));
+
+        assertEquals(
+                Arrays.asList(Collections.singletonList('1'),
+                Collections.singletonList('2'), Collections.singletonList('3'),
+                Collections.emptyList(), Arrays.asList(4, 5, 6)),
+                ListOps.concat(listOfNestedLists));
```

Comment:
You're missing a closing bracket in your `listOfNestedLists` which is why the build is failing. This would be easier to spot if you split your lines by putting each argument to a function on a new line and putting the closing bracket on its own line at the end 🙂 For example:

```
List<List<List<Character>>> listOfNestedLists = Arrays.asList(
    Arrays.asList(
        Collections.singletonList('1'),
        Collections.singletonList('2')
    ),
    Arrays.asList(
        Collections.singletonList('3'),
        Collections.emptyList()
    ),
    Arrays.asList(
        Arrays.asList('4', '5', '6')
    )
);

assertEquals( 
    Arrays.asList(
        Collections.singletonList('1'),
        Collections.singletonList('2'),
        Collections.singletonList('3'),
        Collections.emptyList(),
        Arrays.asList(4, 5, 6)
    ),
    ListOps.concat(listOfNestedLists)
);
```

That should make the code a bit easier to read. Especially for nested function calls like this that can be very useful I've found 🙂